src/components/structures/login/Login.js

```diff
@@ -125,7 +123,7 @@ module.exports = React.createClass({displayName: 'Login',
             // logins so let's skip that for now).
             loginLogic.chooseFlow(0);
         }, function(err) {
-            self._setErrorTextFromError(err);
+            self._setStateFromError(err);
```

Comment:
Oh, I see what you mean. I've added a flag to _setStateFromError to dictate whether it sets loginIncorrect or not.